public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/contactPoint/ContactPointSelector.tsx

```diff
@@ -80,8 +81,7 @@ export function ContactPointSelector({
             render={({ field: { onChange, ref, ...field }, fieldState: { error } }) => (
               <>
                 <div className={styles.contactPointsSelector}>
-                  <Select
-                    {...field}
```

Comment:
there is no need for passing these props  as the `onchange` is the one handling the changes in a custom way, and typescript was complaining about it.